guide/phone_order_delivery_service/phone_order_delivery_service.bal

```diff
@@ -0,0 +1,218 @@
+import ballerina/log;
+import ballerina/http;
+import ballerina/jms;
+
+//Deploying on kubernetes
+
+//import ballerinax/kubernetes;
+// Other imports
+
+// Type definition for a Deliver order
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@kubernetes:Ingress {
+//hostname:"ballerina.guides.io",
+//name:"ballerina-guides-phone_order_delivery_service",
+//path:"/"
+//}
+
+//@kubernetes:Service {
+//serviceType:"NodePort",
+//name:"ballerina-guides-phone_order_delivery_service"
+//}
+
+//@kubernetes:Deployment {
+//image:"ballerina.guides.io/phone_store_service:v1.0",
+//name:"ballerina-guides-phone_order_delivery_service"
+//}
+
+//endpoint http:Listener listener {
+//port:9091
+//};
+
+//@http:ServiceConfig {basePath:"/phonestore1"}
+//service<http:Service> phone_store_service bind listener {
+
+//Deploying on docker
+
+// import ballerinax/docker;
+// Other imports
+
+// Type definition for a  Deliver order
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@docker:Config {
+//registry:"ballerina.guides.io",
+//name:"phone_order_delivery_service",
+//tag:"v1.0"
```

Comment:
Have you tested with docker..? Might need to add baseImage due to https://github.com/ballerinax/docker/issues/117

Reply:
added baseimage